templates/legal/confidentiality-agreement.html

```diff
@@ -498,7 +498,7 @@ <h3>Your details</h3>
                     </li>
                     <li class="p-list__item">
                       <div class="htmlSection" id="tfa_5">
-                        <p>By clicking "I agree" below, I confirm that I have read and understood the terms of the confidentiality agreement, <a href="/legal/data-privacy">privacy policy</a> and <a href="/legal/data-privacy/unilateral-nda">privacy notice</a> and warrant that I have authority to enter into the <a href="https://assets.ubuntu.com/v1/05d55fbc-Canonical+NDA_Online+Form_202008.pdf" target="_blank">confidentiality agreement</a>
+                        <p>By clicking "I agree" below, I confirm that I have read and understood the terms of the confidentiality agreement, <a href="/legal/data-privacy">privacy policy</a> and <a href="/legal/data-privacy/unilateral-nda">privacy notice</a> and warrant that I have authority to enter into the <a href="https://assets.ubuntu.com/v1/05d55fbc-Canonical+NDA_Online+Form_202008.pdf" target="_blank" rel="noopener noreferrer">confidentiality agreement</a>
```

Comment:
I don't think this needs blank as the user have the back button if they need. They are not missing any state.